RoutesApp_iosTests/AlgorithmTests.swift

```diff
@@ -16,15 +16,33 @@ class AlgorithmTests: XCTestCase {
         // MARK: Route 1
         let routePoints1 = TestResources.points1Array
         let stops1 = TestResources.stops1Array
-        line1 = LineRoute(name: "01", id: "01", idLine: "Zy8i3x4nQH0os7TaCizc", line: "Lines/4MhlK4IGLhTL6wcf50xk",
-            routePoints: routePoints1, start: routePoints1[0], stops: stops1, end: routePoints1[1], averageVelocity: 20.5,
-            blackIcon: "", whiteIcon: "", color: "")
+        line1 = LineRoute(name: "01",
+                          id: "01",
+                          idLine: "Zy8i3x4nQH0os7TaCizc",
+                          line: "Lines/4MhlK4IGLhTL6wcf50xk",
```

Comment:
i think we should not use real data in tests at least in ids, replace them for fake ids